README.md

```diff
@@ -7,367 +7,598 @@
+
+📼 **Supertape** provides a CLI that supports running both `CJS` and `ESM` tests:
 
 ```
-Usage: supertape [options] [path]
-Options
-   -h, --help                  display this help and exit
-   -v, --version               output version information and exit
-   -f, --format                use a specific output format - default: progress-bar/tap on CI
-   -r, --require               require module
-   --no-check-scopes           do not check that messages contains scope: 'scope: message'
-   --no-check-assertions-count do not check that assertion count is no more then 1
-   --no-check-duplicates       do not check messages for duplicates
+$ [npx] supertape "tests/test1.js" [options]
 ```
 
-## Environment variables
+If a glob is specified instead of a single file path (e.g. `tests/*.js`), all of the tests results are outputted together.
+
+#### **<u>Command Line Flags</u>:**
+
+#### `--help` (alias: `-h`)
+
+Displays a help prompt with usage information and exits.
+
+#### `--version` (alias: `-v`)
+
+Outputs version information and exits.
+
+#### `--format [format]` (alias: `-f`)
+
+Sets a specific [format](#format-string-since-v381) for outputting test results. When using the CLI, this option must be set to change the format used. Defaults to `progress-bar`.
+
+#### `--require [module]` (alias: `-r`)
+
+Loads a given module before running all tests.
+
+#### `--no-check-duplicates`
+
+Tells 📼 **Supertape** not to check test case messages for duplicates. By default, messages are expected to be unique.
+
+#### `--no-check-assertions-count`
+
+Tells 📼 **Supertape** not to check the number of assertions per test case. By default, only one assertion is allowed.
+
+#### `--no-check-scopes`
+
+Tells 📼 **Supertape** not to check test case messages for proper scoping (i.e. in the form `'scope: message'`). By default, messages are expected to be scoped.
+
+#### **<u>Environment Variables</u>:**
+
+#### `SUPERTAPE_TIMEOUT`
+
+Controls the time (in milliseconds) that a test is allowed to run for before timing out. By default, tests time out after 3 seconds (or `3000` ms).
+
+#### `SUPERTAPE_CHECK_DUPLICATES`
+
+Set to `0` to tell 📼 **Supertape** not to check test case messages for duplicates. By default, messages are expected to be unique.
+
+#### `SUPERTAPE_CHECK_ASSERTIONS_COUNT`
+
+Set to `0` to tell 📼 **Supertape** not to check the number of assertions per test case. By default, only one assertion is allowed.
+
+#### `SUPERTAPE_CHECK_SCOPES`
+
+Set to `0` to tell 📼 **Supertape** not to check test case messages for proper scoping (i.e. in the form `'scope: message'`). By default, messages are expected to be scoped.
+
+#### `CI=1`
 
-- `SUPERTAPE_TIMEOUT` - timeout for long running processes;
-- `SUPERTAPE_CHECK_DUPLICATES` - toggle check duplicates;
-- `SUPERTAPE_CHECK_SCOPES` - check that test message has a scope: `scope: subject`;
-- `SUPERTAPE_CHECK_ASSERTIONS_COUNT` - check that assertion count is no more then 1;
+TODO: disable progress bar?
```

Comment:
When CI detected (we can skip this option since it set by GitHub actions) it disables progress bar int default formatter